src/layouts/Header/Header.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, useState } from "react";
+import classes from "classnames";
+import { Drawer as AntdDrawer } from "antd";
+
+//components
+import Button from "@/components/Button";
+
+//helpers
+import { useWindowDimension } from "@/hooks/useWindowDimension";
+
+//assets
+import LogoImg from "@/assets/icons/LogoImg";
+
+//styles
+import styles from "@/styles/components/header.module.scss"
+
+const Header: FC = () => {
+  const [modalOpen, setModalOpen] = useState(false)
+  const { isMobile } = useWindowDimension();
+
+  const renderNavigationList = () => (
+    <ul className={styles.navList}>
```

Comment:
please, create an array NAVIGATION_LIST = [{title: "about", url:"#"}, {title: "Plans", url:"#"},  {title: "Plans", url:"#", hasButton: true}] and map it to render this list